iib/web/models.py

```diff
@@ -803,10 +811,8 @@ def from_json(cls, kwargs, batch=None):
         request_kwargs = deepcopy(kwargs)
 
         bundles = request_kwargs.get('bundles', [])
-        if (
-            not isinstance(bundles, list)
-            or len(bundles) == 0
-            or any(not item or not isinstance(item, str) for item in bundles)
+        if not isinstance(bundles, list) or any(
+            not item or not isinstance(item, str) for item in bundles
         ):
             raise ValidationError(f'"bundles" should be a non-empty array of strings')
```

Comment:
Empty lists are now allowed. Should this say `"bundles" should be either an empty array or an array of non-empty strings`?

Reply:
Unrelated to your changes, we should remove the `f` prior to the string literal.